src/main/java/com/stripe/model/Account.java

```diff
@@ -144,6 +144,33 @@ public static AccountCollection list(Map<String, Object> params, RequestOptions
   }
   // </editor-fold>
 
+  // <editor-fold desc="persons">
+  /**
+   * Retrieve an account's persons.
```

Comment:
This is a list call, so maybe change this (and the other 2 below) to:
```suggestion
   * List an account's persons.
```

Reply:
Ack, though I did what you did in `Source.java` for source transactions